.drone.yml

```diff
@@ -1,24 +1,93 @@
 ---
 kind: pipeline
-type: docker
-name: build
+name: amd64
 
 platform:
   os: linux
   arch: amd64
 
 steps:
-  - name: test
-    image: golang:1.14.1-buster
-    privileged: true
-    commands:
-      - go test ./...
-    when:
-      event:
-        - pull_request
-        - push
-        - tag
-trigger:
-  event:
-    exclude:
-      - promote
\ No newline at end of file
+- name: build
+  image: rancher/dapper:v0.4.1
+  commands:
+  - dapper ci
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+
+- name: github_binary_release
+  image: plugins/github-release
+  settings:
+    api_key:
+      from_secret: github_token
+    prerelease: true
+    checksum:
+    - sha256
+    checksum_file: CHECKSUMsum-amd64.txt
+    checksum_flatten: true
+    files:
+    - "dist/artifacts/*"
+  when:
+    instance:
+    - drone-publish.rancher.io
+    ref:
+    - refs/head/master
+    - refs/tags/*
+    event:
+    - tag
+
+- name: docker-publish
+  image: plugins/docker
+  settings:
+    dockerfile: package/Dockerfile
+    password:
+      from_secret: docker_password
+    repo: "rancher/drone-docker-image-digests"
+    tag: "${DRONE_TAG}-amd64"
+    username:
+      from_secret: docker_username
+  when:
+    instance:
+    - drone-publish.rancher.io
+    ref:
+    - refs/head/master
+    - refs/tags/*
+    event:
+    - tag
+
+volumes:
+- name: docker
+  host:
+    path: /var/run/docker.sock
+
+---
+kind: pipeline
+name: manifest
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: manifest
```

Comment:
is there even a manifest file? i think we need to do windows and actually build this out